src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -398,7 +400,16 @@ public String getQuestionResultStatisticsHtml(
 
         DecimalFormat df = new DecimalFormat("#.##");
         
-        for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
+        SortedMap<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>(new Comparator<String>() {
```

Comment:
We can leave it as `Map<String, List<Integer>>` instead of `SortedMap<String, List<Integer>>`. In addition, there's no need for the `Comparator` here, the `TreeMap` will automatically sort in ascending order of the `String` keys